### PR TITLE
Spruce up FAQ and How it Works styles

### PR DESCRIPTION
**Pull request by meowwwls**

### CODE REVIEW

File: components/common/textLink.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const TextLink = ({ href, text, children, external, ...props }) => (
         </ChakraLink>
       </Link>
     ) : (
-      <ChakraLink href={href} target='_blank' rel='noopener noreferrer' className={styles.link} {...props}>
+      <ChakraLink href={href} className={styles.link} {...props}>
```
Comment (joelwass) [inline components/common/textLink.js]: We still need this :( we need to try and figure out how to accessibly open a new tab? Is that a thing?

File: pages/faq.js [hunk 2]
```diff
@@ -1,92 +1,123 @@
...
+              src='/images/flossbank_ads_demo.gif'
+              margin='1.5rem auto 0'
+            />
+          </Box>
+          <Divider />
+          <FAQHeading>What package managers do you wrap?</FAQHeading>
+          <Text marginBottom='1.5rem'>
+            We are currently wrapping{' '}
+            <TextLink href='https://www.npmjs.com/' external text='npm' /> and{' '}
+            <TextLink href='https://yarnpkg.com/' external text='Yarn' />. We
```
Comment (joelwass) [inline pages/faq.js:38]: Reason we need the target=blank is we can’t link out to yarn and npm if it redirects the user from the current tab

**Review by joelwass — CHANGES_REQUESTED** [review]

File: components/common/textLink.js (hunk 1 above)
Comment (meowwwls) [reply components/common/textLink.js]: There is, but meeting the visual warning requirement for accessibility is something I'm not sure how we should handle: 
https://www.w3.org/TR/WCAG20-TECHS/G201.html

Screenreader requirement is easy enough to meet. Most links that open in a new tab/window are external links and we could use an icon to indicate as such which is how it's usually done accessibly, but when we're making some internal links open in a new tab/window (are we?), then the icon would be misleading.

Comment (joelwass) [reply components/common/textLink.js]: could we provide an SR only span within the text link that shows when it's an external link?

Comment (meowwwls) [reply components/common/textLink.js]: Yeah that's what I'm going to do; we just need to at some point figure out how to handle the visual indicator to make it fully compliant. :)

**Review by joelwass — APPROVED** [review]

**Pull request merged**